http/go/nethttp/compression_handler_test.go

```diff
@@ -42,24 +42,6 @@ func TestCompressionHandlerAddsContentEncodingHeader(t *testing.T) {
 	assert.Equal(t, contentTypeHeader, "gzip")
 }
 
-func TestCompressionHandlerCompressesRequestBody(t *testing.T) {
```

Comment:
can you expand on why you decided to remove that test please?

Reply:
The test was a false positive and passed when the body had 0 bytes.

Reply:
thanks for the precision here. We should still check the compression is actually performing compression work in case somebody breaks it in the future. hardcoding the expected size of the result would be ok for that purpose I think.